test/contracts.py

```diff
@@ -21,17 +20,69 @@ class BridgeContractConfig:
     next_swap_id: int
 
 DefaultBridgeContractConfig = BridgeContractConfig(
-    cap = "250000000000000000000000000",
-    reverse_aggregated_allowance = "3000000000000000000000000",
-    reverse_aggregated_allowance_approver_cap = "3000000000000000000000000",
-    lower_swap_limit = "1",
-    upper_swap_limit = "1000000000000000000000000",
-    swap_fee = "0",
-    paused_since_block = 18446744073709551615,
-    denom = "atestfet",
-    next_swap_id = 0
+    cap="250000000000000000000000000",
+    reverse_aggregated_allowance="3000000000000000000000000",
+    reverse_aggregated_allowance_approver_cap="3000000000000000000000000",
+    lower_swap_limit="1",
+    upper_swap_limit="1000000000000000000000000",
+    swap_fee="0",
+    paused_since_block=18446744073709551615,
+    denom="atestfet",
+    next_swap_id=0
```

Comment:
Does this change reflect some [PEP](https://peps.python.org/)?

Reply:
I'll put that down to good ol' Jetbrains auto-formatting against my will